mr/src/main/java/org/apache/mahout/cf/taste/impl/similarity/precompute/MultithreadedBatchItemSimilarities.java

```diff
@@ -26,7 +26,7 @@
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicInteger;
 
-import com.google.common.collect.Lists;
+import java.util.ArrayList;
 import com.google.common.io.Closeables;
```

Comment:
Go ahead and do it if u like, I have a big change coming up which fixes
some of that stuff.

On Sun, Apr 5, 2015 at 3:12 AM, Anand Avati <notifications@github.com>
wrote:

> In
> mr/src/main/java/org/apache/mahout/cf/taste/impl/similarity/precompute/MultithreadedBatchItemSimilarities.java
> <https://github.com/apache/mahout/pull/106#discussion_r27775411>:
>
> > @@ -26,7 +26,7 @@
> >  import java.util.concurrent.TimeUnit;
> >  import java.util.concurrent.atomic.AtomicInteger;
> >
> > -import com.google.common.collect.Lists;
> > +import java.util.ArrayList;
> >  import com.google.common.io.Closeables;
>
> Argh, I blindly replaced only collections! I could have removed Closeables
> as well, would be trivial. I'm assuming you will be doing it, based on your
> previous comment.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/mahout/pull/106/files#r27775411>.
>